Application/EdFi.Ods.Api/Security/Profiles/AdminProfileNamesPublisher.cs

```diff
@@ -4,8 +4,9 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 using System;
-using System.Data.Entity;
+using Microsoft.EntityFrameworkCore;
```

Comment:
Reorder using statements to get System.* on top and the rest of the libraries in alphabetic order.